### ISSUE TITLE
File drop for logged in users

### ISSUE THREAD

**Issue by freumichCH** [nextcloud/server#10808]
Hello Dev Team 

As we know, Nextcloud is offering Link-Sharing with a File-Drop only functionality. 
So my question is if its possible to have this feature for Useraccounts  in Nextcloudas well. 

### Steps to reproduce
1. Create a new Folder with the Name "Mailbox" 
2. Switch to Sharing function of the Folder "Mailbox" and add a Person/Group to share it
3. Remove all privileges on the Folder including Read and Download

### Expected behaviour
Entering the Folder shows the Same Screen like the Link-Sharing Dropbox
![bildschirmfoto_2018-08-22_um_18_45_26](https://user-images.githubusercontent.com/22443781/44477695-a8cba600-a63b-11e8-89fd-2c7ed8f9e6d8.png)

### Actual behaviour
Currently does this only work with a Link-Sharing 
![bildschirmfoto_2018-08-22_um_18_49_45](https://user-images.githubusercontent.com/22443781/44477871-3ad3ae80-a63c-11e8-9832-a956d825fc0d.png)

We would love to see this feature in a future Update of Nextcloud and hope that out there some people who will support this idea 

Keep up the good Work! 😊

**Comment by Tealk** [comment]
Hello,

I would like that option too, but it seems to me that it has been rejected?

**Comment by go2sh** [comment]
No, the issue is still open. Only the referenced issue is closed. ;-) I'am also looking for such a feature. I guess its pretty hard to implement as a lot of sharing and front-end code has to be touched, right?! @nickvergessen @skjnldsv

**Comment by nickvergessen** [comment]
Yeah well and the sync clients need to be able to handle it, etc.

**Comment by snoopy0815** [comment]
+1  for this functionality.  Needed for teams, where uploads should be accepted by admins first (e.g. GDPR conformance check)

**Comment by susnux** [comment]
Closing this as a duplicate of https://github.com/nextcloud/server/issues/16515
(Keeping the newer issue as there is already more context available).
